x-pack/plugins/profiling/server/index.ts

```diff
@@ -21,6 +36,7 @@ const configSchema = schema.object({
 });
 
 export type ProfilingConfig = TypeOf<typeof configSchema>;
+export type ConfigType = TypeOf<typeof commonConfigSchema>;
```

Comment:
It appears that since the input variables for the symbolizer and collector packages have a lot in common, then it made sense to abstract this into `ConfigType`. So `ConfigType` is just a type used within the larger `ProfilingConfig`, which captures everything defined using `xpack.profiling.*` in `config/kibana[.dev].yml`.

@cauemarcondes Let me know if that sounds right :)